docs/wordpress.md

```diff
@@ -0,0 +1,18 @@
+# WordPress
+
+The SSO module allows using a WordPress site as the authentication provider for the Platform site. This works via OAuth 2.0, as such the WordPress site you want to use as the authentication provider must be using the [OAuth 2.0 WordPress Plugin](https://github.com/WP-API/OAuth2).
+
+To enabled WordPress SSO, you must create an OAuth 2 application on the WordPress site, and set the Callback URL on the OAuth client to `$site_url/hm-delegated-auth-callback`, for example `https://my-site.altis.dev/hm-delegated-auth-callback`. Once you have published and approved the OAuth application on your WordPress site, take note of the Public Key for the client.
```

Comment:
Is this not a default we can infer?

Reply:
This is about configuring oauth2 on your WordPress site you want to be the provider, not the Platform site, so we don't control anything "over there"